test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -84,13 +89,29 @@
                     return result;
                 }
             };
+        },
+        toMatchIEEE754Number: function (util, customEqualityTesters) {
```

Comment:
This is not a function declaration, this is a function expression.

function declaration:

```
(function () {

    function mydeclaredFunction() {
    };

}());
```

function expression assigned to var:

```
(function () {

    var myFunctionExpression = function () {
    };

}()))
;
```

function expression assigned to property of object:

```
(function () {

    var myObj = {
        myFunctionExpression: function () {
        }
    };

}());
```

> Is labeled function declaration allowed in 'strict' mode in all browser environments?

[ECMAScript permits function statements in exactly two places: directly within the list of statements that make up a program, and directly within the list of statements that make up the contents of a function body](http://whereswalden.com/2011/01/24/new-es5-strict-mode-requirement-function-statements-not-at-top-level-of-a-program-or-function-are-prohibited/)

> Is there a different way to do this without labeled function declaration?

This is not a function declaration / function statement. But I agree that if it was, we shouldn't use that form.